tests/slate/test_darcy_hybridized_mixed.py

```diff
@@ -63,3 +63,8 @@ def test_darcy_flow_hybridization(degree, hdiv_family):
 
     assert sigma_err < 1e-8
     assert u_err < 1e-8
+
+
+if __name__ == '__main__':
+    import os
+    pytest.main(os.path.abspath(__file__))
```

Comment:
Don't bother adding this line.  It's cargo-culted rom something that florian used to do, I keep meaning to remove them.